src/core/Global.ttslua

```diff
@@ -645,6 +646,16 @@ function emptyChaosBag()
   end
 end
 
+-- find all chaos tokens that are on the table to the chaos bag (e.g. sealed on cards)
+function findAllChaosTokens()
```

Comment:
This may be a broader feature for the cleanup helper, but should it clean up Bless/Curse tokens since those are removed at the end of a scenario?

Reply:
Those are already handled (via a call to the bless/curse manager who has the GUIDs stored) before this function is called ;)